### ISSUE TITLE
Create browse digital heritage page

### ISSUE THREAD

**Issue by steve-taylor-wsu** [MukurtuCMS/Mukurtu-CMS#44]
It's been decided we want both a browse page and a browse digital heritage page. Functionally identical, but the DH page is already filtered to DH only.

**Comment by steve-taylor-wsu** [comment]
Right now the "browse" page consists of two parts:
* Browse (path: /browse, route: mukurtu_browse.browse_page, controller: MukurtuBrowseController::content)
* Map Browse (path: /browse/map, route: mukurtu_browse.map_browse_page, controller: MukurtuBrowseByMapController::content)

We need to add analogous pages for a digital heritage browse page that does the same thing, but only shows DH items:
* Browse Digital Heritage (new path:/digital-heritage)
* Browse Digital Heritage on map (new path:/digital-heritage/map)

My first thought on how to implement this would be to reuse (not copy) the browse controllers. The 'mukurtu_browse' view is the thing that actually does the content type filtering. I think we could:
* Create the new routes
* Add a parameter to the routes for which view to use (maybe make it default to 'mukurtu_browse'?)
* Duplicate the mukurtu_browse view (e.g., mukurtu_digital_heritage_browse), change it to display only DH items
* Configure the new route definitions parameters to use that view
* Modify the controllers to no longer use the hardcoded 'mukurtu_browse' view and use the route parameter

I *think* that's better than straight up duplicating everything? If it turns out the above is messy, we can talk about duplicating.

Also, once you have it working, do me a favor and on a test site check the behavior in the case where the entire digital heritage content type has been removed. If it does something bad (e.g., crashes) how should we address? Check for the existence of the DH type, redirect to the regular browse page if it doesn't?

**Comment by steve-taylor-wsu** [comment]
Oh a word of warning, you can't blindly search/replace 'mukurtu_browse' in the controllers. There are non-views (at the very least a theme) also named 'mukurtu_browse' :sweat_smile:. You'll need to read the code and make sure only to replace the view references.

**Comment by nick-deer** [comment]
Alright I got a first attempt working! I tested out deleting the DH content type and the site crashed in two places. First, it crashed after I tried navigating to the Mukurtu Dashboard (message: "view.mukurtu_manage_all_content.mukurtu_manage_content does not exist"). Then it crashed when I navigated directly to the new route, /digital-heritage (crashed on looking for 'field_coverage' in the controller query). 

For the second crash case, I agree with your suggestion to check that DH exists first and to redirect to the normal browse page on failure. I'm not sure how to handle the first crash case.
